ably/realtime_conn.go

```diff
@@ -85,10 +85,16 @@ func (c *Connection) dial(proto string, u *url.URL) (proto.Conn, error) {
 	query := u.Query()
 	query.Add("heartbeats", "true")
 	u.RawQuery = query.Encode()
+	// (RTN14c) we are setting timeout to connect.
+	timeout := c.opts.RealtimeRequestTimeout
+	if timeout == 0 {
+		// use defined default value
+		timeout = defaultOptions.RealtimeRequestTimeout
```

Comment:
If you rebase this on top of latest `integration/1.2`, this isn't necessary, as the default values get applied [earlier](https://github.com/ably/ably-go/blob/43dafd3246e3ef14b3091c8b3ea45ddd9867f11b/ably/rest_client.go#L139).

Reply:
Thanks . Done. I did a `git push -f` after a rebase so can't reference the commit.